src/components/Tooltip/index.js

```diff
@@ -0,0 +1,57 @@
+import React, {PropTypes} from 'react'
+
+const passStrengthValues = [10, 20, 25, 30, 33, 34, 40, 50, 60, 70, 75, 80, 90, 100]
+
+const Tooltip = ({children}) => {
+  const ttClasses = 'relative db br2 w-100 mt4 pa3 sans-serif navy bg-white eh-input-tooltip'  
+  return (
+    <div className={ttClasses}>
+      {children}
+    </div>
+  )
+}
+Tooltip.propTypes = {
+  children: React.PropTypes.node.isRequired
+}
+
+/**
+ * Password-specific Tooltip
```

Comment:
I agree that comments should only be used if there is something really confusing or whatever - but even then I'd put it in the readme over inline.